src/tracer/src/config/config_loader.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
```

Comment:
i would change this filename to `config.rs` only, as it doesn't load anything. It's just the object declaration